driver/build.gradle

```diff
@@ -43,7 +43,11 @@ clirr {
 
 tasks.withType(Checkstyle) {
     // needed so the Javadoc checks can find the code in other modules
-    classpath = files(project(':driver-core').sourceSets.main.output, sourceSets.main.output);
+    classpath = files(project(':driver-core').sourceSets.main.output, sourceSets.main.output)
+    reports {
```

Comment:
it makes it much easier to read the checkstyle errors.  Weirdly it works for the bson section even though its not defined there - I'll move to the common area.